scripts/chassismodulesshow

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/python
+"""
+    Script to show chassis modules status.
+"""
+from __future__ import print_function
+
+import argparse
+
+from tabulate import tabulate
+from swsssdk import SonicV2Connector
+from natsort import natsorted
+
+
+header = ['Module', 'Instance', 'Type', 'Status']
+
+CHASSIS_MODULE_TABLE_NAME = 'CHASSIS_MODULE_TABLE'
+MODULE_FIELD_NAME = 'name'
+INSTANCE_FIELD_NAME = 'instance'
+TYPE_FIELD_NAME = 'type'
+STATUS_FIELD_NAME = 'status'
+
+class ChassisModuleShow(object):
+    def __init__(self):
+        self.db = SonicV2Connector(host="127.0.0.1")
+        self.db.connect(self.db.STATE_DB)
+
+    def show(self):
+        keys = self.db.keys(self.db.STATE_DB, CHASSIS_MODULE_TABLE_NAME + '*')
+        if not keys:
+            print('Chassis-Module Not detected\n')
+            return
+
+        table = []
+        for key in natsorted(keys):
+            key_list = key.split('|')
+            if len(key_list) != 2: # error data in DB, log it and ignore
+                print('Warn: Invalid key in table CHASSIS_MODULE_TABLE: {}'.format(key))
+                continue
+
+            data_dict = self.db.get_all(self.db.STATE_DB, key)
+            name = data_dict[MODULE_FIELD_NAME]
+            instance = data_dict[INSTANCE_FIELD_NAME]
+            type = data_dict[TYPE_FIELD_NAME]
+            status = data_dict[STATUS_FIELD_NAME]
```

Comment:
Per today's sync-u:
a) In show platform or rather revised show platform chassis 'DB' and 'command output', represent a card via <card_type><instance_number> and store corresponding platform slot_id alongside.
Also, introduce 'Description' field relevant to card.
e.g.  line card inserted in slot 3 as LINECARD3  slot_id as 7 
        fabric card inserted in slot 3 as FABRICCARD3 and slot_id as 9 
        control card inserted in slot 0 as CONTROLCARD0 and slot_id as 0

b) Further, can we look into shortening these names as LC3 , FC3, CC0.
This would give room for Description filed to have enough space/length to display relevant/meaningful information

Reply:
As discussed in the review, we are starting with SUPERVISOR0, LINE-CARD0, FABRIC-CARD0. Slot number is a separate field and can be vendor specific.